core/src/bootstrap/lwm2m_bootstrap.c

```diff
@@ -237,6 +237,7 @@ bool Lwm2mBootstrap_BootStrapInit(Lwm2mContextType * context, const char ** conf
     bool success = true;
     int i = 0;
 
+    memset(bootStrapQueue, 0, sizeof(bootStrapQueue));
```

Comment:
The compiler will initialise this to zero anyway as it's declared static. but better safe than sorry ;)